client/jetpack-cloud/sections/partner-portal/controller.tsx

```diff
@@ -13,6 +12,7 @@ import BillingDashboard from 'calypso/jetpack-cloud/sections/partner-portal/prim
 import CompanyDetailsDashboard from 'calypso/jetpack-cloud/sections/partner-portal/primary/company-details-dashboard';
 import DownloadProducts from 'calypso/jetpack-cloud/sections/partner-portal/primary/download-products';
 import InvoicesDashboard from 'calypso/jetpack-cloud/sections/partner-portal/primary/invoices-dashboard';
+import IssueLicenseV2 from 'calypso/jetpack-cloud/sections/partner-portal/primary/issue-license';
```

Comment:
Can we rename this to `IssueLicenseV2` not that we have moved this to `/issue-license`?